include/misc/statistics.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <forward_list>
+
+#include <seqan3/io/sam_file/input.hpp>
+
+/*! \brief Returns a list of pairs, where the first integer is a random chromosome and the second is a random position.
+ *        The list is sorted by default.
+ *  \param sample_size - The number of positions to generate.
+ *  \param header - The header from the SAM file input.
+ * 
+ *  \return Returns a sorted forward_list of pairs of reference chromosome IDs and positions.
+ */
+std::forward_list<std::pair<int32_t, int32_t>> get_random_positions(uint64_t sample_size, seqan3::sam_file_header<std::deque<std::string>> & header)
+{
+    std::forward_list<std::pair<int32_t, int32_t>> output{};
+    if (sample_size == 0)
+    {
+        output.push_front(std::pair<int32_t, int32_t>{-1, -1});
```

Comment:
```suggestion
        output.push_front(std::pair<int32_t, int32_t>{-1, -1});
        return output;
```

this makes it clear that this is a special case.